tests/installation/bootloader_hyperv.pm

```diff
@@ -145,12 +145,16 @@ sub run {
 
     my ($winserver, $winver, $vmguid);
 
-    $winver = $svirt->get_cmd_output("cmd /C ver");
-    if (grep { /Microsoft Windows \[Version 6.3.*\]/ } $winver) {
-        $winserver = 2012;
+    $winver = $svirt->get_cmd_output("$ps (Get-WmiObject Win32_OperatingSystem).Version");
+    $winver =~ s/^\s+|\s+$//g;
+    if ($winver eq '6.3.9600') {
```

Comment:
I guess the last minor numbers 9600 depends on which win2k12 minor versions is used? If yes, suggest to only match major version number 6.3, so as to survive win host updates.

Reply:
Windows server 2012R2 uses Windows NT 6.3, if I remember correctly. But Window 2016 and 2019 are really hard to tell apart by version.

Reply:
First, there are two servicing channels for Windows Server release: LTSC(Long-Term Servicing Channel) and SAC(Semi-Annual Channel). Suppose we used the formal release build which should get from LTSC, and windows updates will not change the version/build number unless upgrading to a main release. From my perspective, we can just add and check those main release numbers that's enough. 

References:
[1] https://docs.microsoft.com/en-us/windows-server/get-started/windows-server-release-info
[2] https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19
[3] https://gaijin.at/en/infos/windows-version-numbers

Reply:
Among QA (product and maintenance), there are two major groups of hyperv SUTs now in openqa which maybe installed with different versions, eg windows 2019 , some use the version updated at 2020 Dec, while the other uses one updated later, and both will likely to update to latest updates(not sure whether version gets changed, needs your test result of local host update after registered with new license )

Reply:
Probably we can put a set of build numbers in an array for matching whether it's a valid windows server build for running tests. Will update and resubmit a PR for review.